Makefile

```
@@ -0,0 +1,57 @@
+.PHONY: $(MAKECMDGOALS)
+
+clean:
+	@rm -rf tests/tmp
+
+cleantests:
+	@rm -rf tests/tmp/.git
+	@rm -rf tests/tmp/empty
+	@rm -rf tests/tmp/CHANGELOG.md
```

```suggestion
	@rm -f tests/tmp/CHANGELOG.md
```
I think we wouldn't want to touch it if it's a directory... Hm.

---

I just blindly use `-rf` all the time since `rm` is fine with it. But good suggestion anyway.